openeo/local/processing.py

```diff
@@ -34,7 +43,15 @@ def load_local_collection(*args, **kwargs):
     elif '.nc' in collection.suffixes:
         data = xr.open_dataset(kwargs['id'],chunks={},decode_coords='all').to_array(dim='bands') # Add decode_coords='all' if the crs as a band gives some issues
     elif '.tiff' in collection.suffixes or '.tif' in collection.suffixes:
-        data = rioxarray.open_rasterio(kwargs['id']).rename({'band':'bands'})
+        data = rioxarray.open_rasterio(kwargs['id'],chunks={},band_as_variable=True)
+        for d in data.data_vars:
+            data_attrs_lowercase = [x.lower() for x in data[d].attrs]
+            data_attrs_original  = [x for x in data[d].attrs]
+            data_attrs = dict(zip(data_attrs_lowercase,data_attrs_original))
+            if 'description' in data_attrs_lowercase:
+                data = data.rename({d:data[d].attrs[data_attrs['description']]})
```

Comment:
That looks quite convoluted. If I understand correctly, you just want to find a "description" attr (if any) in a case insensitive manner. 
I think this should do it:
```python
descriptions = [v for k, v in data[d].attrs if k.lower() == "description"]
if descriptions:
    data = data.rename({d: data[d].attrs[descriptions[0]])
```

Reply:
thanks, I'll try it soon!